### PR TITLE
Api 3044 add pdf data to upload report

### PR DESCRIPTION
**Pull request by cshupp1**
## Description of change
Adding in a rake task that works via:
rake vba_documents:inspect_pdf[/path/to/pdfs]

Adding in support for endpoint:
http://localhost:3000/services/vba_documents/v1/uploads/report
to include additional data as follows:

"uploaded_pdf": {
                    "total_documents": 3,
                    "total_pages": 6,
                    "content": {
                        "page_count": 4,
                        "dimensions": {
                            "height": 8.5,
                            "width": 11.0
                        },
                        "oversized_pdf": false,
                        "attachments": [
                            {
                                "page_count": 1,
                                "dimensions": {
                                    "height": 27.11,
                                    "width": 45.33
                                },
                                "oversized_pdf": true
                            },
                            {
                                "page_count": 1,
                                "dimensions": {
                                    "height": 8.5,
                                    "width": 11.0
                                },
                                "oversized_pdf": false
                            }
                        ]
                    }

## Original issue(s)
https://vajira.max.gov/browse/API-3044
https://vajira.max.gov/browse/API-3050

## Things to know about this PR

rake task run against files pulled from prod.

rspecs created to test endpoint logic.
rspec ./modules/vba_documents/spec/jobs/upload_processor_spec.rb
rspec ./modules/vba_documents/spec/request/v1/reports_request_spec.rb 
rspec ./modules/vba_documents/spec/lib/multipart_parser_spec.rb

### CODE REVIEW

File: modules/vba_documents/app/swagger/vba_documents/document_upload/status_attributes_swagger.rb [hunk 1]
```diff
@@ -37,7 +36,14 @@ class StatusAttributesSwagger
...
 
           property :updated_at do
-            key :description, 'The last time the submission was updated'
+            key :description, 'The last time the submission was updated at Greg'
+            key :type, :string
+            key :format, 'date-time'
+            key :example, '2018-07-30T17:31:15.958Z'
+          end
+
+          property :Cris do
```
Comment (bastosmichael) [inline modules/vba_documents/app/swagger/vba_documents/document_upload/status_attributes_swagger.rb:45]: Did you mean to commit this?

**Review by bastosmichael — CHANGES_REQUESTED** [review]
Make sure to attach the separate Migration PR as well as there are some things here that look like they weren't intended to be committed.

File: modules/vba_documents/app/swagger/vba_documents/document_upload/status_attributes_swagger.rb (hunk 1 above)
Comment (cshupp1) [reply modules/vba_documents/app/swagger/vba_documents/document_upload/status_attributes_swagger.rb]: Nope, THanks for catching.  Removed.

**Review by bastosmichael — APPROVED** [review]
LGTM, let's make sure we have another ticket for updating the release notes https://developer.va.gov/release-notes/benefits#benefits

**Pull request merged**